client/src/util/actionHelpers.ts

```diff
@@ -74,8 +76,10 @@ export const doFetch = async (url: any, init = {}) => {
 /*
 Wrapper to perform an async fetch and JSON decode response.
 */
-// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any -- - FIXME: disabled temporarily on migrate to TS.
-export const doJsonRequest = async (url: any, init = {}) => {
+export const doJsonRequest = async (
```

Comment:
should be a generic so caller can specify expected return type of the json object.

Ie, caller can do something like:
```
  const schema = doJsonRequest<Schema>("/path/to/schema");
```

Reply:
I left doJsonRequest as an unknown return type and instead used the generic in `fetchJson`

Reply:
(this was because `doJsonRequest` is only called by `fetchJson`)

Reply:
not sure I understand the rationale for leaving doJsonRequest untyped.  You could have more flexibility, and exactly the same default typing if you declare it as:
```
export const doJsonRequest = async<T = unknown> (...) : Promise<T> {...}
```

this lets you remove the cast from fetchJson.  (sometimes casts are a hint that you may be able to improve things....)

Reply:
Adding generic SGTM!

```suggestion
export const doJsonRequest = async <T = unknown>(
```